# MoveJoint.srv

# Request
float64[] point1
float64[] point2
float64 velocity
float64 acceleration
---
# Response
control_msgs/JointTrajectoryControllerState joint_state